### ISSUE TITLE
Potential problem/confusion with anthropogenic heat setup in URBPARM.TBL

### ISSUE THREAD

**Issue by epn09** [wrf-model/WRF#2081]
**Describe the bug**

In URBPARM.TBL we have:

```
#
# AH:  Anthropogenic heat [ W m{-2} ]
#      (sf_urban_physics=1)
#

AH:  20.0, 50.0, 90.0

#
# Anthropogenic Heating diurnal profile.
#

AHDIUPRF: 0.16 0.13 0.08 0.07 0.08 0.26 0.67 0.99 0.89 0.79 0.74 0.73 0.75 0.76 0.82 0.90 1.00 0.95 0.68 0.61 0.53 0.35 0.21 0.18
```

* Usually, I would expect `AH` to be the daily mean anthropogenic heat and `AHDIUPRF` to be a factor that convert daily average to hourly value.
* However, in the example given in URBPARM.TBL, `AH` is the daily *maximum* anthropogenic heat. Indeed, if we multiply 20.0 with `AHDIUPRF (0.16, 0.13, 0.08, ..., 0.18)` and take the daily average, only 11.1 W/m^2 of anthropogenic heat is emitted in a day.
* If AH is intended to be the daily mean anthropogenic heat, then the `AHDIUPRF` should has the mean of 1. So, the example above should be normalized to `0.29 0.23 0.14 0.13 0.14 0.47 1.21 1.78 1.60 1.42 1.33 1.31 1.35 1.37 1.48 1.62 1.80 1.71 1.22 1.10 0.95 0.63 0.38 0.32`

**To Reproduce**
N/A

**Expected behavior**
* I want to confirm if the original intention was to make `AH` the daily *mean* or *maximum* anthropogenic heat.
  * If it's mean: `AHDIUPRF` should be fixed.
  * If it's maximum: It is somewhat unusual (to me) that the maximum value to be given as input but it's not a bug.
* In either case, I could not find any documentation or user guide mentioning the intended behavior, so it might be better to document it somewhere.

**Screenshots**
N/A

**Attachments**
N/A

**Additional context**
N/A

**Comment by weiwangncar** [comment]
@cenlinhe Can you comment on this post? Thanks!

**Comment by cenlinhe** [comment]
I do not have an answer for this as I do not know the history of AH implementation. Maybe @barlage knows.

**Comment by barlage** [comment]
I don't know the original intention as it is implemented in WRF, but if one looks at the original Salamanca paper: 

> For the UCM a diurnal profile of AH was added to the sensible heat flux (hereafter this simulation is referred as UCM+AH) with **peak values of 90, 50, and 20 W m−2** for the commercial or industrial (COI), high-intensity residential (HIR), and low-intensity residential (LIR) urban classes, respectively.

That said, one of the advantages of this being in a parameter table is that it is easy for users to change, as long as the parameter table makes it clear how the values are being used. URBPARM.TBL should probably be updated to make this more clear, e.g., AH should be peak daily anthro heat and AHDIUPRF should be made clearer that it is the multiplicative factor for hourly AH.

**Comment by dudhia** [comment]
I agree, peak value would have been my interpretation when looking at the
diurnal factors, but it should be made clearer I guess.

**Comment by cenlinhe** [comment]
@xuelingbo @doan-van, feel free to leave any comments or suggestions here. This is about the anthropogenic heat in SLUCM.

**Comment by epn09** [comment]
@barlage Thanks for investigating. Maybe it's enough to explicitly mention that AH is the maximum value.

**Comment by xuelingbo** [comment]
I agree that `AH` is the peak value.
